react/components/ProgressBar.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+function ActiveBar () {
+  return <span className="progress-bar progress-bar--active">| </span>;
```

Comment:
What is the `|` doing here?

Reply:
It looks as if its seeding the progress visually, is that correct?

Reply:
Yes, I am rendering the `|`s for the progress bars